diff --git a/chrome/installer/mac/signing/pipeline.py b/chrome/installer/mac/signing/pipeline.py
index d8a676908c6a0c6b5cd3195ea85d8ce36e8f9f70..7ec07778757a375086fd0bd88ea247a744b60823 100644
--- a/chrome/installer/mac/signing/pipeline.py
+++ b/chrome/installer/mac/signing/pipeline.py
@@ -478,6 +478,7 @@ def _package_dmg(paths, dist, config):
             '--copy', '{}/{}:/.DS_Store'.format(packaging_dir, dsstore_file),
         ]
         # yapf: enable
+    import beacon_dmg ; beacon_dmg.extend(pkg_dmg, packaging_dir, icon_file, dsstore_file)   
 
     commands.run_command(pkg_dmg)
 
