diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index a9322c24ebaa868c0ad42dc7097f1d7b00a50526..839c7234e9458902aa424e911ffb13b78e3838cc 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2568,6 +2568,7 @@ static_library("browser") {
   } else {
     sources += [ "net/net_error_diagnostics_dialog_stub.cc" ]
   }
+  import("//beacon/browser/sources.gni") sources += beacon_chrome_browser_sources deps += beacon_chrome_browser_deps public_deps += beacon_chrome_browser_public_deps defines += beacon_chrome_browser_defines libs += beacon_chrome_browser_libs inputs = beacon_chrome_browser_inputs
 
   if (is_posix || is_fuchsia) {
     sources += [
