diff --git a/chrome/installer/mac/signing/parts.py b/chrome/installer/mac/signing/parts.py
index d4247ed4fe0004ccce8552d15e3a6c33e1cc136c..9955567937f13e2ce2576bba03b59ba51709d36b 100644
--- a/chrome/installer/mac/signing/parts.py
+++ b/chrome/installer/mac/signing/parts.py
@@ -123,6 +123,7 @@ def get_parts(config):
         'libswiftshader_libEGL.dylib',
         'libswiftshader_libGLESv2.dylib',
         'libvk_swiftshader.dylib',
+        'libbeacon.dylib',
     ]
     if config.is_chrome_branded():
         dylibs.append('liboptimization_guide_internal.dylib')
