diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index 253ea5174c65841acb3106ad6e36dbce357c8fb4..9f0c27513abbd060ad081c8e5739a3a8d477e000 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -813,6 +813,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   };
 
   if (!url.SchemeIs(content::kChromeUIScheme))
+  if (!url.SchemeIs(content::kBeaconUIScheme))
     return false;
 
   return base::Contains(kill_hosts, url.host_piece());
