diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 4a50fdd3dc6535a67f10235c9076c40feccb8704..b09d08b3c0c79ae4b6a40b80ab644e4c0b9de390 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -518,6 +518,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//beacon/common:constants_configs" ]
 }
 
 component("chrome_features") {
