diff --git a/components/page_info/page_info_ui.cc b/components/page_info/page_info_ui.cc
index dc41aa1f9ad79cafd47d53a372a641783835ba79..ba349b6d3b0a7a5588ff48a314db078eea58aec9 100644
--- a/components/page_info/page_info_ui.cc
+++ b/components/page_info/page_info_ui.cc
@@ -514,7 +514,7 @@ PageInfoUI::GetSecurityDescription(const IdentityInfo& identity_info) const {
                                            IDS_PAGE_INFO_MIXED_CONTENT_DETAILS,
                                            SecurityDescriptionType::CONNECTION);
         default:
-
+          BEACON_ADD_PAGE_INFO_DETAILS
           auto description = CreateSecurityDescription(
               SecuritySummaryColor::GREEN, IDS_PAGE_INFO_SECURE_SUMMARY,
               base::FeatureList::IsEnabled(
