diff --git a/components/os_crypt/keychain_password_mac.mm b/components/os_crypt/keychain_password_mac.mm
index 5589310e2e1f41a6a97e77bb57a7a71cd09a18be..db3aaf577abe1444a9fae14701b36bbe9fe10475 100644
--- a/components/os_crypt/keychain_password_mac.mm
+++ b/components/os_crypt/keychain_password_mac.mm
@@ -64,12 +64,14 @@ std::string AddRandomPasswordToKeychain(const AppleKeychain& keychain,
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetServiceName() {
+  BEACON_GET_SERVICE_NAME
   static KeychainNameContainerType service_name(kDefaultServiceName);
   return *service_name;
 }
 
 // static
 KeychainPassword::KeychainNameType& KeychainPassword::GetAccountName() {
+  BEACON_GET_ACCOUNT_NAME
   static KeychainNameContainerType account_name(kDefaultAccountName);
   return *account_name;
 }
