diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 88890346ff12752883a0cc51adbbb8ecbb9f4f54..95b69f015dac4875f9b06dfb53c44658ca1aeac9 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -278,6 +278,8 @@ source_set("browser") {
       "//chromeos/services/machine_learning/public/mojom",
     ]
   }
+  deps += [ "//beacon/services/trust" ]
+
 
   public_deps = [
     "//base",
@@ -1289,6 +1291,8 @@ source_set("browser") {
     "network_service_client.h",
     "network_service_instance_impl.cc",
     "network_service_instance_impl.h",
+    "//beacon/content/browser/trust_service_instance_impl.cc",
+    "//beacon/content/browser/trust_service_instance_impl.h",
     "notification_service_impl.cc",
     "notification_service_impl.h",
     "notifications/blink_notification_service_impl.cc",
