diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index 535a207ee6317422139b671ea7115218e299a76d..2589e7150c57e7e3b61530a6775674a06d873141 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1200,6 +1200,7 @@ class DnsTransactionImpl : public DnsTransaction,
     if (!DNSDomainFromDot(hostname_, &labeled_hostname))
       return ERR_INVALID_ARGUMENT;
 
+    BEACON_DISABLE_SUFFIX_SEARCH
     if (hostname_.back() == '.') {
       // It's a fully-qualified name, no suffix search.
       qnames_.push_back(labeled_hostname);
@@ -1360,6 +1361,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BEACON_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
