diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index edccf23b09ad7d9f49ea18f2f832b08b60cb57e8..52b4c776c37807d558f9cb2a05df38fb6951d3e1 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1202,6 +1202,7 @@ class DnsTransactionImpl : public DnsTransaction,
     if (!DNSDomainFromDot(hostname_, &labeled_hostname))
       return ERR_INVALID_ARGUMENT;
 
+    BEACON_DISABLE_SUFFIX_SEARCH
     if (hostname_.back() == '.') {
       // It's a fully-qualified name, no suffix search.
       qnames_.push_back(labeled_hostname);
@@ -1362,6 +1363,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BEACON_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
