diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index a929bbd2a3f975a7f1477b80e511c1c4cbb5adcf..80ba7f64c78d6df4988479eaa9b70179fe6078ee 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -271,6 +271,7 @@ component("network_service") {
     "//url",
   ]
 
+  deps += [ "//beacon/services/trust/client" ]
   if (is_linux || is_chromeos) {
     deps += [ ":network_sandbox_hook" ]
   }
