diff --git a/services/network/BUILD.gn b/services/network/BUILD.gn
index ab096d2a11bae02ee8e09b55dc49199c0c09ab4a..1cfdd7b3b5071ca69564e0a7e747ae2efc177af8 100644
--- a/services/network/BUILD.gn
+++ b/services/network/BUILD.gn
@@ -267,6 +267,7 @@ component("network_service") {
     "//url",
   ]
 
+  deps += [ "//beacon/services/trust/client" ]
   if (is_linux || is_chromeos) {
     deps += [ ":network_sandbox_hook" ]
   }
